meta.json

```diff
@@ -0,0 +1,19 @@
+{
+    "module_id": "viam:torchvision",
+    "visibility": "public",
+    "url": "https://github.com/viam-labs/torchvision",
+    "description": "This vision service provides a wrapper around torchvision functionality.",
+    "models": [
+      {
+        "api": "rdk:service:vision",
+        "model": "viam:vision:torchvision"
+      }
+    ],
+    "build": {
+        "setup": "./setup.sh",
+        "build": "./build.sh", 
+        "path": "dist/archive.tar.gz", 
+        "arch": ["linux/amd64", "linux/arm64"] 
```

Comment:
I just realized we have darwin/arm64 runner, so please add "darwin/arm64" to this